rust/src/index/vector/kmeans.rs

```diff
@@ -32,12 +32,17 @@ fn train_kmeans_faiss(
     Ok(model.centroids.into())
 }
 
-/// Fallback implementation of KMeans.
-fn train_kmeans_fallback() -> Result<Float32Array> {
+/// A fallback default implementation of KMeans, if not accelerator found.
+fn train_kmeans_fallback(
+    _array: &Float32Array,
+    _dimension: usize,
+    _k: u32,
+    _max_iter: u32,
+) -> Result<Float32Array> {
     todo!()
```

Comment:
Should this panic just so it doesn't pass silently?

Reply:
oh, `todo()` actually raises `not implemented` error. that's why I disabled the test in `Dataset.rs`, unfortunately.